<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1084</width>
    <height>1087</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="resetRequirementButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>700</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Requirement</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textChoiceCost">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>460</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Cost</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="choices">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>1001</width>
      <height>141</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="existingRequirements">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>540</y>
      <width>1001</width>
      <height>141</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textChoiceDescription">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>460</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSentenceConversation_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>500</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Requirements</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addRequirementButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>700</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Requirement</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSentenceConversation_4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add Choice</string>
    </property>
   </widget>
   <widget class="QComboBox" name="requirementKeyComboBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>700</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>MED</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SUR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PSY</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>REP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MAI</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>INT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SEC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PAC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>REL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BIO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AST</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AIS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ANY</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelSentenceConversation_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Choices</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addChoiceButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>460</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Choice</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textRequirementValue">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>700</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Requirement Value</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="eventIdx">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>150</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="placeholderText">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="selectedFilePath">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Selected File</string>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textDescription">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>511</width>
      <height>51</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Descripton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectFileButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reduceButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>150</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="increaseButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>150</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>120</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Event Index</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="isCrisisCheckbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>IS CRISIS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateJsonButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>1010</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Update JSON</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>1020</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Info Message:</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusMessage">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>1020</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="choiceTrigram">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>460</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>MED</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ENG</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SCI</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FAI</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DEF</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="endIdxButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>150</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>End</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetEffectButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>950</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Effect</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textArgument">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>950</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>argument</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addEffectButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>950</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Effect</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSentenceConversation_6">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>750</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Effects</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="existingEffects">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>790</y>
      <width>1001</width>
      <height>141</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textMethodName">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>950</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>MethodName</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
